### DISCUSSION TITLE
QQ音乐歌曲地址api失效

### DISCUSSION THREAD

**Discussion by Lizc521** [xizeyoupan/Meting-API#32] [Q&A]
我通过自己部署的api能够获取歌单没问题，可以获取到
![image](https://github.com/user-attachments/assets/fc556f5c-0ef6-471b-8104-5a251c3b8706)

下面的歌词链接也没问题，但是如果点击URL，会被重定向到一个新的地址，且无法直接访问
![image](https://github.com/user-attachments/assets/e09e0766-c62c-4ba7-a17b-a1bb608054ce)
![image](https://github.com/user-attachments/assets/1f30cac5-fa45-4108-a17b-3cc76adee838)

**Comment by xizeyoupan** [comment]
qq音乐要部署在国内ip

**Reply by Lizc521** [reply]
我是参考了readme部署在国内的
![image](https://github.com/user-attachments/assets/4025d78a-b4a5-4c28-957c-cb1b6a8383c0)

**Reply by Lizc521** [reply]
我刚刚查看了您的接口部署在大陆填的是否，且可以播放QQ音乐，我填的是，播放不了QQ音乐，网易是正常的，难道是这里的区别？
![image](https://github.com/user-attachments/assets/9bdf8b4f-529e-4e35-ab79-b002a20a527c)

**Reply by Lizc521** [reply]
果然，我把这里改成否之后，QQ音乐正常了
